README.md

```diff
@@ -116,6 +116,7 @@ def saml_settings
   settings.idp_sso_target_url             = "https://app.onelogin.com/trust/saml2/http-post/sso/#{OneLoginAppId}"
   settings.idp_slo_target_url             = "https://app.onelogin.com/trust/saml2/http-redirect/slo/#{OneLoginAppId}"
   settings.idp_cert_fingerprint           = OneLoginAppCertFingerPrint
+  settings.idp_cert_fingerprint_alg       = "http://www.w3.org/2000/09/xmldsig#sha1"
```

Comment:
It would be nice if this were fully spelled out: `idp_cert_fingerprint_algorithm`.